frontend/configuration.properties.js

```diff
@@ -309,6 +309,14 @@
     // a tooltip containing the full name will be added to the component.
     maxHierarchicalSegmentLength: 0,
   },
+  
+  // These properties configure the tags/options shown for each of the list search results
+  SearchResultTags: {
+    // Whether to show an option to add/view comments for a search result. False by default.
+    comments: false,
+    // Specify the table field for the comment documents in the index. 'comments' by default.
+    commentsTable: 'comments'
```

Comment:
@nikhilj13, what would be the impact of specifying `comments` as the default for someone who updates to this version and isn't currently using comments?